pkg/appmanager/routing.go

```diff
@@ -198,17 +198,16 @@ func processIngressRules(
 }
 
 func httpRedirectIRule(port int32) string {
-	iRuleCode := fmt.Sprintf(`
-	when HTTP_REQUEST {
+	iRuleCode := fmt.Sprintf(
+		`when HTTP_REQUEST {
        HTTP::redirect https://[getfield [HTTP::host] ":" 1]:%d[HTTP::uri]
     }`, port)
 
 	return iRuleCode
 }
 
 func sslPassthroughIRule() string {
-	iRuleCode := `
-when CLIENT_ACCEPTED {
+	iRuleCode := `when CLIENT_ACCEPTED {
 	TCP::collect
 }
 
```

Comment:
gofmt doesn't like indenting multiline strings, so it has to be manually done. I think it would look better to properly indent this large string block to look more natural within the function. Same with the httpRedirectIrule above.

Reply:
Unless that is going to cause these same whitespace errors.

Reply:
I pushed a formatting change for the larger irule. The redirect irule is so small that gofmt keeps putting it back the way it shows here.

Reply:
I think you pushed a branch to upstream.

Reply:
Oops, fixed.